src/test/bswap_protobuf_tests.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2016 The Bitcoin Core developers
```

Comment:
I'd prefer moving this one to the qt unit tests, instead of adding a qt-specific unit test to the core unit tests.